guides/plugins/plugins/framework/rate-limiter/add-rate-limiter-to-api-route.md

```diff
@@ -0,0 +1,213 @@
+# Add Rate Limiter to API Route
+
+## Overview
+
+In this guide you'll learn how to secure API routes with a rate limit to reduce the risk against bruteforce.
+
+## Prerequisites
+
+This guide is built upon both the [Plugin base guide](../../plugin-base-guide.md) as well as the [Dependency injection](../../plugin-fundamentals/dependency-injection.md) guide.
+
+Furthermore you need an existing API route, to create a new one, head over to our [Add store API route](../store-api/add-store-api-route.md) guide.
+
+## Configuring rate limit
+
+{% hint style="danger" %}
+Do not remove the default rate limits, otherwise it will throw an exception due to they are used in the core.
+
+If you want to disable them, just set `enabled` to `false` in 
+`<platform root>/src/Core/Framework/Resources/config/packages/shopware.yaml` under `shopware.api.rate_limiter`.
```

Comment:
Never touch core files

The users should create a `config/packages/shopware.yaml` with the override content